src/App.tsx

```diff
@@ -33,6 +37,11 @@ const App = () => {
   const { enableAutoPageviews } = plausible
   enableAutoPageviews()
 
+  useEffect(() => {
+    fetch(`${api.BACKEND_URL}/csrf/`, { credentials: "include" }).then(() => {
+    }).catch((e) => console.error(e));
+  });
+  
```

Comment:
Do we need this?

Reply:
Yes. The reason why we never need this before in this project was because all of the requests we did were only `GET`.

However, with `POST` and `PUT` requests we need to send the CSRF token to the backend. 

In order to send the CSRF token, we need to get it first.

Reply:
Right!